tools/flaky-test-reporter/config/config.yaml

```diff
@@ -24,49 +24,49 @@ jobConfigs:
     repo: serving
     type: postsubmit
     slackChannels:
-      - name: networking
+      - name: net-istio
         identity: CA9RHBGJX
```

Comment:
Please also change the identity, which you can get by "right click the channel -> Additional options -> Copy link".

The same for other channels below as well.

Reply:
Done